app/controllers/employees_controller.rb

```diff
@@ -11,27 +11,33 @@ def new
   end
 
   def show
-
+    @employees = Employee.find(params[:id])
   end
 
   def create
-    data = params.require(:employee)
-    employee = Employee.new
-    employee.nameEmployee = data["nameEmployee"]
-    employee.emailEmployee = data["emailEmployee"]
-    employee.positionEmployee = data["positionEmployee"]
-    employee.attendanceDate = data["attendanceDate"]
```

Comment:
You should use snake case with your employee attributes
Example:
name_employee

Reply:
Sure, but i need to modify some files including my tables, we will planned to do that on a new issue